### PR TITLE
Change file permission

### PR DESCRIPTION
**Pull request by maipbui**
Restrict file permission access.

### CODE REVIEW

**Comment by maipbui** [comment]
@a-barboza Could you help review this PR?

**Comment by a-barboza** [comment]
> @a-barboza Could you help review this PR?

@maipbui Do you want to check if 0600 will cause issue if read permission is required by non-root users? Sometimes, processes without privileges need to be able to read these files, I think. In that case 0644 might be better. In any case I do not have any objections to the stats file having 0600 permissions.

**Comment by maipbui** [comment]
> > @a-barboza Could you help review this PR?
> 
> @maipbui Do you want to check if 0600 will cause issue if read permission is required by non-root users? Sometimes, processes without privileges need to be able to read these files (PAM configuration files), I think. In that case 0644 might be better. In any case I do not have any objections to the stats file having 0600 permissions.

Yes, I do want to check and verify the change. Could you help verify?

File: scripts/hostcfgd [hunk 1]
```diff
@@ -1016,7 +1016,7 @@ class PasswHardening(object):
         # Use rename(), which is atomic (on the same fs) to avoid empty file
         with open(PAM_PASSWORD_CONF + ".tmp", 'w') as f:
             f.write(pam_passwh_conf)
-        os.chmod(PAM_PASSWORD_CONF + ".tmp", 0o644)
+        os.chmod(PAM_PASSWORD_CONF + ".tmp", 0o600)
```
Comment (qiluo-msft) [inline scripts/hostcfgd:1019]: >0o600 [](http://example.com/codeflow?start=45&length=5)

I checked the permission for `/etc/pam.d/common-password` is 644. So suggest chmod to the same permission on .tmp file.

File: scripts/aaastatsd [hunk 2]
```diff
@@ -151,7 +151,7 @@ class RadiusStatistics:
                 os.unlink(stats_file)
         else:
             open(stats_file, 'a').close()
-            os.chmod(stats_file, 0o666)
+            os.chmod(stats_file, 0o600)
```
Comment (qiluo-msft) [inline scripts/aaastatsd:154]: >0o600 [](http://example.com/codeflow?start=33&length=5)

We need to check original file permission.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**